docs/integrations/sources/snapchat-marketing.md

```diff
@@ -123,8 +123,9 @@ Snapchat Marketing API has limitations to 1000 items per page.
   <summary>Expand to review</summary>
 
 | Version | Date       | Pull Request                                             | Subject                                                                        |
-| :------ | :--------- | :------------------------------------------------------- | :----------------------------------------------------------------------------- |
-| 0.6.2   | 2024-15-22 | [38574](https://github.com/airbytehq/airbyte/pull/38574) | Update authenticator package                                                   |
+|:--------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------------------------------|
+| 1.0.0   | 2024-06-20 | [39507](https://github.com/airbytehq/airbyte/pull/39507) | Migrate to low-code                                                            |
```

Comment:
Migrate to low-code CDK and add incremental functionality to Organizations

Reply:
updated